igvc_ws/src/igvc_nav/src/mt_dstar_node.py

```diff
@@ -135,7 +164,7 @@ def make_map(c_space):
     # Reset the path
     path = None
 
-    robot_pos = (100 - int((curEKF.x - map_reference[0]) / GRID_SIZE), 100 - int((curEKF.y - map_reference[1]) / GRID_SIZE))
+    robot_pos = (100, 100)
```

Comment:
Is this just being used to initialize the map? I don't see robot_pos being changed anywhere which seems like a problem.

Reply:
That file is full of messes from the old way we wanted to handle map generation. `robot_pos` was originally going to be how we know where the robot is relative to the *stiched* map, but then we decided stiching is too hard